lib/core/ui/context_menu_item.dart

```diff
@@ -2,14 +2,11 @@ import 'package:flutter/material.dart';
 
 class ContextMenuItem extends StatelessWidget {
   const ContextMenuItem({
-    @required this.title,
-    @required this.icon,
-    @required this.onTap,
-    Key key,
-  })  : assert(title != null),
-        assert(icon != null),
-        assert(onTap != null),
-        super(key: key);
+    required this.title,
+    required this.icon,
+    required this.onTap,
+    Key? key,
+  }) : super(key: key);
 
   final String title;
   final IconData icon;
```

Comment:
- Satır 27'de icon `nullable` olabileceğinden dolayı `icon ?? Icons.warning` şeklinde düzenlenmiş olabilir.
- Satır 6'da bu şekilde düzenlenebilir https://github.com/flutterturkey/festival_flutterturkiye_org/pull/108/files#r814005591

```suggestion
  final IconData? icon;
```

Reply:
null gelme olasılığı orda yok, gereksiz kullanım uyarısı vardı ondan sildim.